http_service/pkg/http_user_service/service.go

```diff
@@ -0,0 +1,36 @@
+package httpuserservice
+
+import (
+	"context"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+
+	"github.com/jcromanu/final_project/http_service/pkg/entities"
+)
+
+type HTTPService interface {
```

Comment:
As Neider mentioned before, the interfaces must be defined where are used (in this case I suppose in the endpoint layer)

Reply:
ok